core/src/test/java/kafka/testkit/KafkaClusterTestKit.java

```diff
@@ -227,8 +231,12 @@ public KafkaClusterTestKit build() throws Exception {
                     setupNodeDirectories(baseDirectory, node.metadataDirectory(), Collections.emptyList());
                     BootstrapMetadata bootstrapMetadata = BootstrapMetadata.
                         fromVersion(nodes.bootstrapMetadataVersion(), "testkit");
+                    MetaPropertiesEnsemble metaPropertiesEnsemble = new MetaPropertiesEnsemble.Loader().
+                        setLoadMissingBehavior(LoadMissingBehavior.EXCEPTION).
+                        addMetadataLogDir(node.metadataDirectory()).
```

Comment:
LoadMissingBehavior.EXCEPTION is not defined anywhere.

Reply:
Yes, this was left over from some earlier code, sorry. Removed.